django_lti_tool_provider/models.py

```diff
@@ -57,3 +61,36 @@ def send_lti_grade(self, grade):
         ))
 
         return outcome
+
+    @classmethod
+    def get_or_create_by_parameters(cls, user, authentication_manager, lti_params):
+        """
+        Filters out OAuth parameters
```

Comment:
@Kelketek nit: this docstring would make sense on `LTIView.lti_param_filter`, but this method does much more than that. Needs updating :)